src/index.ts

```diff
@@ -482,9 +482,9 @@ export default class NestedList {
     classes.push(styleClass);
 
     // since tag is either 'ol' or 'ul' we can safely cast it to HTMLOListElement | HTMLUListElement
-    return Dom.make(tag, [this.CSS.wrapper, ...classes]) as
-      | HTMLOListElement
-      | HTMLUListElement;
+    return Dom.make(tag, [this.CSS.wrapper, ...classes], {
+      contentEditable: (!this.readOnly).toString()
```

Comment:
We can't just add `contenteditable` to the wrapper without preventing default events (and re-implementing them manually) because user can break a tool's DOM by pressing delete, for example.